Bottom: 06900af70bba616c5b99cee41b3b61e710c525d2
Top:    ee546fd6a868de0891389fa6377f9bc5153f8fdc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:45:33 +0000

find.c: Pass flags to sub-functions without using static variable "TheFlag".


---

diff --git a/src/find.c b/src/find.c
index 2b7f2b4..9bd44b8 100644
--- a/src/find.c
+++ b/src/find.c
@@ -287,7 +287,6 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-static int TheFlag = FOUNDFLAG;
 static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
@@ -967,12 +966,12 @@ LookupPVConnectionsToPVList (void)
 
 struct lo_info
 {
-  Cardinal layer;
-  LineType line;
-  PadType pad;
-  ArcType arc;
-  PolygonType polygon;
-  RatType rat;
+//  Cardinal layer;
+//  LineType line;
+//  PadType pad;
+//  ArcType arc;
+//  PolygonType polygon;
+//  RatType rat;
   jmp_buf env;
 };
